install/kubernetes/helm/istio/charts/mixer/templates/config.yaml

```diff
@@ -430,9 +429,8 @@ spec:
       destination_service_namespace: destination.service.namespace | "unknown"
       request_protocol: api.protocol | context.protocol | "unknown"
       response_code: response.code | 200
+      grpc_response_status: response.grpc_status | 0
       response_flags: context.proxy_error_code | "-"
-      permissive_response_code: rbac.permissive.response_code | "none"
-      permissive_response_policyid: rbac.permissive.effective_policy_id | "none"
```

Comment:
why are these removed? cardinality issue?

Reply:
For months I've been asking if these are used, and have yet to find anyone that is actually using these dimensions. I *thought* we were not carrying them forward into v2 (though the code now says otherwise).

@mandarjog objections to removing?

Reply:
@liminw @yangminzhu can we remove these?

Reply:
fwiw, kialki does not use these.

Reply:
Yes please remove, @yangminzhu last chance to object.

Reply:
Sorry I missed this PR. It should be fine to remove because the new authorization policy doesn't support the permissive mode but I'm not sure if it's used by anyone in the old RBAC policy for now. cc @liminw